vscode-extension/src/aiConfigEditor.ts

```diff
@@ -305,10 +306,10 @@ export class AIConfigEditorProvider implements vscode.CustomTextEditorProvider {
     });
 
     // TODO: saqadri - stderr is very noisy for some reason (duplicating INFO logs). Figure out why before enabling this.
-    // startServer.stderr.on("data", (data) => {
-    //   this.extensionOutputChannel.error(this.prependMessage(data, document));
-    //   console.error(`server stderr: ${data}`);
-    // });
+    startServer.stderr.on("data", (data) => {
+      this.extensionOutputChannel.info(this.prependMessage(data, document));
+      console.error(`server stderr: ${data}`);

```

Comment:
Based on the comment, should this be enabled yet? Also, info level but logs to error?

Reply:
Good catch, will fix at top of stack